### ISSUE TITLE
soul frames only work once

### ISSUE THREAD

**Issue by danielrossyamitrano** [Chocohead/Binnie-Patcher#32]
It seems that the soul frames only do their job once. After the frame starts showing its durability, the chance of successful mutation drops back to normal (or so it seems). I've noticed this issue when I was trying to breed imperial bees. After 6 generations the mutation didn't happen (I just got hybrids). But as soon as I replaced the frames with new ones, I got the imperial bee that I was looking for.

**Comment by Chocohead** [comment]
Might be related to #10, the mutation modifications seem to be flaky at best.

**Comment by danielrossyamitrano** [comment]
Yes, I also thought that the two issues may be related. I think it is a problem with the vanilla mutation system, not with the soulframes or the mutator per se.
